.travis.yml

```diff
@@ -88,6 +88,7 @@ install:
     pip uninstall opencv-contrib-python -y;

```

Comment:
This is also a possible typo by me. Kindly remove `pip uninstall opencv-contrib-python -y;` and add `pip uninstall opencv-python -y;`.